Makefile.power

```diff
@@ -10,7 +10,7 @@ USE_OPENMP = 1
 endif
 
 ifeq ($(CORE), POWER10)
-COMMON_OPT += -Ofast -mcpu=power10 -mtune=power10 -mvsx  -fno-fast-math
+COMMON_OPT += -Ofast -mcpu=power10 -mtune=power10 -mvsx -mmma  -fno-fast-math
```

Comment:
We dont need to add -mmma as -mcpu=power10 takes care of enabling mma options by default.
If we add the binutils check, we don't need this change here.

Reply:
Yes, this was only my initial reaction to the issue ticket - the actual problem must be that this line has no effect when COMMON_OPT is passed in via the `make` command line.